CONTRIBUTING.md

```diff
@@ -141,6 +141,33 @@ to email the patch to <gap@gap-system.org>.  You can refer to
 [this tutorial](https://burzalodowa.wordpress.com/2013/10/05/how-to-send-patches-with-git-send-email/)
 on how to do this.
 
+## Reviewing code contributions
+
+When reviewing code, you can use the following list as a checklist.
+
+### Before you even run the code (see [here](https://lornajane.net/posts/2015/code-reviews-before-you-even-run-the-code))
+
+* Is it clear what feature / fix the contribution adresses?
+* Does it relate to exactly one issue?
+* Do the changed files look sensible? Use `git log --stat` to discover
+possibly unintentional changes to files.
+* Do the commit messages look good? Should some commits be squashed / broken up?
+* Eyeball the diff for
+  * Large commented / unused sections of code
+  * Strange variable or function names 
+  * Duplicate code
+
+
+### Look at the code itself
```

Comment:
A lot of the previous section already deals with looking at the code itself, doesn't it? It's not so clear to me why this new sections starts exactly here...